themes/digital.gov/src/scss/new/shortcodes/_do-dont-table.scss

```diff
@@ -0,0 +1,21 @@
+@use "uswds-core" as *;
+
+.dg-do-dont-table {
+  thead tr th:first-child {
+    @include u-border-top("05", "green-40v", "solid");
+    @include u-color("green-50v");
+  }
+
+  thead tr th:last-child {
```

Comment:
BEM name for the `thead` and/or the `th` inside please.

Reply:
BEM name added